cmd/ui/src/components/GraphEvents.tsx

```diff
@@ -26,47 +26,39 @@ import {
     resetCamera,
 } from 'src/ducks/graph/utils';
 import { bezier } from 'src/rendering/utils/bezier';
-import { getNodeRadius } from 'src/rendering/utils/utils';
+import { DRAG_DELAY, MOUSE_BUTTON_PRIMARY, getNodeRadius, preventAllDefaults } from 'src/rendering/utils/utils';
 import { sequentialLayout, standardLayout } from 'src/views/Explore/utils';
 
 export interface GraphEventProps {
-    onDoubleClickNode?: (id: string) => void;
     onClickNode?: (id: string) => void;
     onClickEdge?: (id: string, relatedFindingType?: string | null) => void;
     onClickStage?: () => void;
-    edgeReducer?: (edge: string, data: Attributes, graph: AbstractGraph) => Attributes;
     onRightClickNode?: (event: SigmaNodeEventPayload) => void;
     showNodeLabels?: boolean;
     showEdgeLabels?: boolean;
 }
 
 export const GraphEvents = forwardRef(function GraphEvents(
     {
-        onDoubleClickNode,
         onClickNode,
         onClickEdge,
         onClickStage,
         onRightClickNode,
-        edgeReducer,
         showNodeLabels = true,
         showEdgeLabels = true,
     }: GraphEventProps,
     ref
 ) {
-    const { selectedItem } = useExploreSelectedItem();
+    const { cancelHighlight, highlightedItem, selectedItem } = useExploreSelectedItem();
```

Comment:
The GraphItemInformationPanel is linked to the selectedItem state. In order to keep this panel open while still allowing for the highlighted item to be unhighlighted, a separate state is used. The state is added to this hook because it needs to react to edge selection which is not triggered at this level.